### PR TITLE
Add Mistral example

### PR DESCRIPTION
**Pull request by kunal-vaishnavi**
### Description

This PR adds Mistral as an example in the `examples` folder.

### Motivation and Context

Mistral is one of the supported models in ONNX Runtime GenAI.

### CODE REVIEW

**Review by yufenglee — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Users can call a high level `generate()` method, or run each iteration of the mo
 
 ## Coming very soon
 
-* Support for the Mistral and Whisper model architectures
+* Support for the Whisper model architectures
```
Comment (RyanUnderhill) [inline README.md:33]: We do have Whisper support in there now, but it just doesn't handle multiple audio chunks

**Comment by yufenglee** [comment]
Thanks!

File: README.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ Install onnxruntime-genai.
 ```python
 import onnxruntime_genai as og
 
-model=og.Model(f'models/microsoft/phi-2', device_type)
+model = og.Model(f'models/microsoft/phi-2', device_type)
 
 tokenizer = model.create_tokenizer()
```
Comment (RyanUnderhill) [inline README.md:54]: This should be:
`tokenizer = og.Tokenizer(model)`

Comment (RyanUnderhill) [reply README.md:54]: (while you're modifying this file)

File: README.md [hunk 3]
```diff
@@ -60,11 +60,11 @@ prompt = '''def print_prime(n):
 
 tokens = tokenizer.encode(prompt)
 
-params=og.SearchParams(model)
+params = og.SearchParams(model)
 params.max_length = 200
```
Comment (RyanUnderhill) [inline README.md:64]: This should be:

`params.set_search_options({"max_length":200})`

**Pull request merged**